notNeededPackages.json

```
@@ -936,6 +936,18 @@
             "sourceRepoURL": "https://github.com/malijs/mali",
             "asOfVersion": "0.9.2"
         },
+        {
+            "libraryName": "mali-compose",
+            "typingsPackageName": "mali-compose",
+            "sourceRepoURL": "https://github.com/malijs/mali-compose",
+            "asOfVersion": "0.1.2"
```

The latest version published is [1.1.0](https://www.npmjs.com/package/@types/mali-compose). Please update the version here to a value higher than the published version.
```suggestion
            "asOfVersion": "1.2.0"
```